fw/http_parser.c

```diff
@@ -5551,8 +5551,8 @@ do {									\
 	__fsm_n += step_inc;						\
 	goto match_meth;						\
 } while (0)
-#define __MK_METH_UNKNOWN()						\
-do { req->method = _TFW_HTTP_METH_UNKNOWN; } while (0)
+#define __MK_METH_INCOMPLETE()						\
+do { req->method = _TFW_HTTP_METH_INCOMPLETE; } while (0)
```

Comment:
I'd propose to not to go with the single line macro and just directly assign the method to `req->method` in the code